pkgs/development/python-modules/icontract/default.nix

```diff
@@ -30,6 +33,16 @@ buildPythonPackage rec {
     hash = "sha256-QyuegyjVyRLQS0DjBJXpTDNeBM7LigGJ5cztVOO7e3Y=";
   };
 
+  patches = [
+    # fix needs an old version of deal to comply with the tests
+    # see https://github.com/Parquery/icontract/issues/244
+    (fetchpatch {
+      url = "https://github.com/Parquery/icontract/commit/7cfdba4116cf2faa9f86c626023c886aee292312.patch";
```

Comment:
The commit that actually made it into upstream master was https://github.com/Parquery/icontract/commit/6b1c10f16d5d55d22b8207fc334001cc03c09be1 (again, presumably a rebase-merge)